.github/workflows/php.yml

```diff
@@ -0,0 +1,66 @@
+name: PHP tests
+on: [push, pull_request]
+jobs:
+    # Check there is no syntax errors in the project
+    php-linter:
+        name: PHP Syntax check 5.6|7.2|7.3
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.0.0
+
+            - name: PHP syntax checker 5.6
+              uses: prestashop/github-action-php-lint/5.6@master
+
+            - name: PHP syntax checker 7.2
+              uses: prestashop/github-action-php-lint/7.2@master
+
+            - name: PHP syntax checker 7.3
+              uses: prestashop/github-action-php-lint/7.3@master
+
+    # Check the PHP code follow the coding standards
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.0.0
+
+            - name: Run PHP-CS-Fixer
+              uses: prestashopcorp/github-action-php-cs-fixer@master
```

Comment:
Please don't use this one, it's not taking in consideration what you specify in your composer.lock

Reply:
Fixed